### PR TITLE
Leg class

### PR DESCRIPTION
**Pull request by prateek-balani**
Improved on the legibility classifier by adding better regularisation and loss function on Resnet 34 using SoccerNet training data

### CODE REVIEW

**Review by GorgeousGrace — APPROVED** [review]

**Review by aamirahmed2004 — APPROVED** [review]
Looks good, the accuracy seems to be better than what they reported!
